### PR TITLE
Feature sql db

### PR DESCRIPTION
**Pull request by Azman234**
updated DB details and created fields for DB information

### CODE REVIEW

**Review by Azman234 — COMMENTED** [review]
reviewed the code 

-most of the changes were changes to less developed code which only had a few lines in them 
-the code has been changed and progressed in the feature sub branches

**Review by matthew-luthi — APPROVED** [review]
Search.php has hard coded values for article properties but we can change this later. Otherwise everything else looks good.

File: conf/DBLogin.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php 
+
+
+// Include file with sql details
```
Comment (dioStaxis) [inline conf/DBLogin.php]: do we need both this and sqlinfo.php?

**Review by dioStaxis — APPROVED** [review]

File: conf/DBLogin.php (hunk 1 above)
Comment (Azman234) [reply conf/DBLogin.php]: it will be deleted as it is redundant

**Pull request merged**